### ISSUE TITLE
Wrong winding order on toposimplify

### ISSUE THREAD

**Issue by Fil** [topojson/topojson-simplify#7]
`countries.topo.json`, winding order OK (from a visual check)

`toposimplify -s 0.00001 < countries.topo.json > simplified.topo.json`

=> the simplified file has a certain number of countries (e.g. ZAR-South Africa) which span the globe.

- files at https://gist.github.com/Fil/a6714566a7326e23dc59854dfd90b143

- test at http://blockbuilder.org/Fil/2d2b41d2851d7bbc7421b56bb2b7f4d5

**Comment by Fil** [comment]
It fails with world-atlas/50m.json too, and in the browser:
http://blockbuilder.org/Fil/51b8b3f4aeb56138c671d690ef967a27

**Comment by Fil** [comment]
Happy to report that it works well with topojson-client@2.1.0 and topojson-simplify@2.0.0
http://blockbuilder.org/Fil/070d8e9df7093121a5d2e9851e9b2e48
with
```
<script src="https://unpkg.com/topojson-client@2"></script>
<script src="https://unpkg.com/topojson-simplify@2"></script>
```

**Comment by mbostock** [comment]
This appears to be caused by a5b9e65b63af2473dd48ffd9717eb2383e66625a. Revert to the earlier definition of topojson.sphericalTriangleArea fixes the problem. (However, I am doubtful that reverting this definition is the correct fix to this issue.)

**Comment by mbostock** [comment]
Fixed in 3.0.2.
